Advanced-user-guide.md

```diff
@@ -36,6 +36,38 @@ If you would like this script to support another Linux distro, you should create
 
 Below are some common configurations which you may want to modify:
 
+### Add docker credential
+Edit the following fields of `config/vsphere.pkrvars.hcl` file to add docker crendential.
+```hcl
+common_ray_docker_image = "rayproject/ray:latest"
+common_ray_docker_repo = "docker.io"
+common_ray_docker_username = "<your-docker-username>"
+common_ray_docker_password = "<your-docker-password>"
+```
```

Comment:
This part is not "advanced", and usually is mandatory, please put into README, thanks.

Reply:
In README, it didn't mentioned any detailed configuration. And there is sentence "For more detailed guidance, check the Advanced-user-guide.md" in README.